install-steps.md

```diff
@@ -562,6 +588,9 @@ oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --t
         baremetal: {}
     platform:
       baremetal:
+        # If you are using cached images you need to configure the following properties (bootstrapOSImage and clusterOSImage) so the installer gets the images from the cache
```

Comment:
Add a link that points to the cacheing section of the existing doc.

Reply:
Done.